app/core/decorators/__init__.py

```
@@ -0,0 +1,3 @@
+from .performance import measure_performance
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)